Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/fedora:39 AS builder
+FROM registry.fedoraproject.org/fedora:40 AS builder
```

Comment:
Do this but also on the next "FROM" line.

Reply:
missed that one. Thanks for spotting that.